client/src/components/Login.tsx

```diff
@@ -252,7 +252,8 @@ const LoginWithoutI18n = (props: withI18nProps) => {
     <>
       <Trans render="h1">You are logged in</Trans>
       <Trans render="h2">
-        <JFCLLocaleLink to={home}>Search for an address</JFCLLocaleLink> to add to Building Updates
+        <JFCLLocaleLink to={home}>Search for an address</JFCLLocaleLink> to add to your Building
+        Updates, or visit your <JFCLLocaleLink to={account.settings}>Account</JFCLLocaleLink>.
```

Comment:
maybe clarifying what actions can be done on the account page could be helpful

```suggestion
        Updates, or visit your <JFCLLocaleLink to={account.settings}>account</JFCLLocaleLink> page to manage subscriptions.
```

Reply:
Yeah good idea, that's not really clear otherwise